lib_test/test_common.ml

```diff
@@ -110,7 +110,14 @@ let create: (module Irmin.S_MAKER) -> [`String | `Json] -> (module Irmin.S) =
     in
     let module S = Irmin.Basic(B)(C) in (module S)
 
-type kind = [`Mem | `Fs | `Git | `Http of kind]
+type kind = [`Mem
+	    | `Fs
+	    | `Git
+	    | `Link_FS_Chunck_Krypto
+	    | `Link_MEM_Chunck_Krypto
+	    | `Link_FS_Chunck
+	    | `Link_MEM_Chunck
+	    | `Http of kind]
```

Comment:
does it make sense ti test the chunk and the crypto module in isolation as well (ie. without using the link layer)?